src/nspawn/nspawn-mount.c

```diff
@@ -600,6 +610,47 @@ int mount_custom(
         return 0;
 }
 
+// Retrieve existing subsystems. This function is called in a new cgroup namespace.
+static int get_controllers(Set *subsystems)
+{
```

Comment:
nitpicks: please following coding style: /* foo */ → // foo, and place opening bracket on same line as function name, see CODING_STYLE.